.gitignore

```diff
@@ -5,6 +5,7 @@ learn
 
 # python auto generated files
 __pycache__
+.vscode/*
```

Comment:
you can use `git rm --cached -r .vscode` from project root to remove the already committed dir from the repository. just fyi only. 😄